kimchi/src/verifier_index.rs

```diff
@@ -109,10 +111,17 @@ pub struct VerifierIndex<G: KimchiCurve> {
     #[serde(bound = "PolyComm<G>: Serialize + DeserializeOwned")]
     pub chacha_comm: Option<[PolyComm<G>; 4]>,
 
-    // Range check gates polynomial commitments
     #[serde(bound = "PolyComm<G>: Serialize + DeserializeOwned")]
     pub range_check_comm: Option<[PolyComm<G>; range_check::gadget::GATE_COUNT]>,
 
+    // Foreign field modulus
+    //#[serde(bound = "Option<ForeignElement<G::ScalarField, LIMB_COUNT>>: Serialize + DeserializeOwned")]
```

Comment:
Remove comment and change to `#[serde(bound = "Option<BigUint>: Serialize + DeserializeOwned")]`?